ndk-glue/src/lib.rs

```diff
@@ -14,6 +14,26 @@ use std::ptr::NonNull;
 use std::sync::{RwLock, RwLockReadGuard};
 use std::thread;
 
+#[macro_export]
+macro_rules! ndk_glue {
```

Comment:
May be it will the better to implement _proc_macro_ instead.
So using attribute macros the usage may looks like the follow:
```rust
#[ndk::entry]
fn main() {
// ....
}
```

Reply:
that may be a good improvement, but I'm getting some PR fatigue, can we leave it for a follow up?

Reply:
Of course, don't worry :)